### PR TITLE
Add signed WORD and DWORD types

### PR DESCRIPTION
**Pull request by critto-bg**
I did not add signed byte yet. It's implemented differently and needs more testing.

### CODE REVIEW

File: infinity/format.py [hunk 1]
```diff
@@ -412,9 +416,9 @@ def get_datum_size (self, d, obj=None):
 
         if type == 'BYTE':
             size = 1
-        elif type == 'WORD':
+        elif type in ['WORD', 'SWORD']:
```
Comment (edheldil) [inline infinity/format.py]: Please use tuples " ... in ('WORD', 'SWORD') ...." for this purpose.

File: infinity/format.py [hunk 2]
```diff
@@ -412,9 +416,9 @@ def get_datum_size (self, d, obj=None):
...
             size = 2
-        elif type == 'DWORD':
+        elif type in ['DWORD', 'SDWORD']:
```
Comment (edheldil) [inline infinity/format.py]: tuples, please

File: infinity/format.py (hunk 1 above)
Comment (lynxlynxlynx) [inline infinity/format.py]: Why does it matter?

Comment (critto-bg) [inline infinity/format.py]: Updated the code

**Pull request merged**